cirq/sim/mux.py

```diff
@@ -18,7 +18,7 @@
 
```

Comment:
Unrelated: why is this file call mux? Something related to mulitplexer?

Reply:
+1 for maybe a better name?

Reply:
It multiplexes over the simulators. It's a mux.

... Okay fine, it's not a very good name. Terrible, even.

Reply:
At the top of this file it says `Sampling/simulation methods that delegate to appropriate simulators` which makes sense cause that's what a mux means. I think this is fine if we explain the file's name in that description.